velox/type/DecimalUtil.h

```diff
@@ -86,6 +86,12 @@ class DecimalUtil {
         value);
   }
 
+  // Check if the precision can represent the value.
+  template <typename T>
+  FOLLY_ALWAYS_INLINE static bool valueInRange(T value, uint8_t precision) {
```

Comment:
A function with this name already exists. Perhaps, rename this function to avoid confusion: valueInPrecisionRange.